src/transaction.rs

```diff
@@ -63,13 +70,25 @@ impl Transaction {
     }
 
     pub fn verify(&self) -> bool {
-        self.sig.verify(&self.from, &self.get_sign_data()) && self.plan.verify(self.tokens)
+        self.sig.verify(&self.from, &self.get_sign_data())
+            && self.data.plan.verify(self.data.tokens)
     }
 }
 
+#[cfg(test)]
+pub fn test_tx() -> Transaction {
```

Comment:
Move this into mod tests below and give it a name that's a verb. Just a create_ prefix would be fine.

Reply:
that would make mod tests public